src/log.cpp

```diff
@@ -310,16 +313,29 @@ void Logger::logToOutputs(LogLevel lev, const std::string &combined,
 //// *LogOutput methods
 ////
 
-void FileLogOutput::open(const std::string &filename)
+void FileLogOutput::setFile(const std::string &filename)
 {
-	m_stream.open(filename.c_str(), std::ios::app | std::ios::ate);
+	actionstream << "Log messages are saved to " << filename << std::endl;
```

Comment:
Wouldn't `infostream` or `verbosestream` suffice? I for one wouldn't want this to be printed every time I run Minetest. :)

Reply:
This message is not saved in the log because it is printed before opening the file.
If you don't want it to appear in the terminal, you can disable logging to debug.txt. Why would you use debug.txt when you always open minetest in a tty and directly see the log messages?